tests/core/test_lock_store.py

```diff
@@ -384,3 +387,42 @@ async def test_redis_lock_store_with_valid_prefix(monkeypatch: MonkeyPatch):
     with pytest.raises(LockError):
         async with lock_store.lock("some sender"):
             pass
+
+
+def test_tracker_store_endpoint_config_loading(endpoints_path: Text):
```

Comment:
Default fixture file does not have field username.
Create a new fixture file and use that.

Reply:
Ah, I've added the username field in the existing yaml file. I hope that also works?